SonarQube.Client.Tests/SonarQubeService_GetTaintVulnerabilitiesRequest.cs

```diff
@@ -0,0 +1,648 @@
+		},
+		{
+			""organization"": ""default-organization"",
+			""key"": ""com.sonarsource:citytour2019-java:src/main/java/foo/security/crypto/KeyPairUtil.java"",
+			""uuid"": ""AW0p2QjJ-y65ELkujuRX"",
+			""enabled"": true,
+			""qualifier"": ""FIL"",
+			""name"": ""KeyPairUtil.java"",
+			""longName"": ""src/main/java/foo/security/crypto/KeyPairUtil.java"",
+			""path"": ""src/main/java/foo/security/crypto/KeyPairUtil.java""
+		},
+		{
+			""organization"": ""default-organization"",
+			""key"": ""com.sonarsource:citytour2019-java:src/main/java/foo/security/injection/BusinessThingsUtils.java"",
+			""uuid"": ""AW0p2QjJ-y65ELkujuRJ"",
+			""enabled"": true,
+			""qualifier"": ""FIL"",
+			""name"": ""BusinessThingsUtils.java"",
+			""longName"": ""src/main/java/foo/security/injection/BusinessThingsUtils.java"",
+			""path"": ""src/main/java/foo/security/injection/BusinessThingsUtils.java""
+		},
+		{
+			""organization"": ""default-organization"",
+			""key"": ""com.sonarsource:citytour2019-java:src/main/java/foo/security/injection/CommandInjectionVulnerability.java"",
+			""uuid"": ""AW0p2QjJ-y65ELkujuRL"",
+			""enabled"": true,
+			""qualifier"": ""FIL"",
+			""name"": ""CommandInjectionVulnerability.java"",
+			""longName"": ""src/main/java/foo/security/injection/CommandInjectionVulnerability.java"",
+			""path"": ""src/main/java/foo/security/injection/CommandInjectionVulnerability.java""
+		}
+	],
+	""rules"": [
+		{
+			""key"": ""java:S3330"",
+			""name"": ""Creating cookies without the \""HttpOnly\"" flag is security-sensitive"",
+			""lang"": ""java"",
+			""status"": ""READY"",
+			""langName"": ""Java""
+		},
+		{
+			""key"": ""javasecurity:S3649"",
+			""name"": ""Database queries should not be vulnerable to injection attacks"",
+			""lang"": ""java"",
+			""status"": ""READY"",
+			""langName"": ""Java""
+		},
+		{
+			""key"": ""javasecurity:S2076"",
+			""name"": ""OS commands should not be vulnerable to command injection attacks"",
+			""lang"": ""java"",
+			""status"": ""READY"",
+			""langName"": ""Java""
+		},
+		{
+			""key"": ""java:S4426"",
+			""name"": ""Cryptographic keys should be robust"",
+			""lang"": ""java"",
+			""status"": ""READY"",
+			""langName"": ""Java""
+		}
+	]
+}
+");
+
+            var result = await service.GetTaintVulnerabilitiesAsync("shared", CancellationToken.None);
+
+            messageHandler.VerifyAll();
+
+            // should return only 2 taint out of 4 vulnerabilities
+            result.Count.Should().Be(2);
+
+            var taint1 = result[0];
+
+            taint1.RuleId.Should().Be("javasecurity:S2076");
+            taint1.Message.Should().Be("Refactor this code to not construct the OS command from tainted, user-controlled data.");
+            taint1.FilePath.Should().Be("src\\main\\java\\foo\\security\\injection\\CommandInjectionVulnerability.java");
+            taint1.Hash.Should().Be("1703916771e6abb765843e62a76fcb5a");
+            taint1.Line.Should().Be(17);
```

Comment:
FYI the API also returns TextRange field but we don't currently store it. The sink is in the locations as well so it's probably not that critical.